### PR TITLE
filter series of updates

### PR DESCRIPTION
**Pull request by djnadackal**
pending one is the favourite button tweaks and favourite saved filters

### CODE REVIEW

File: filter/example/src/App.js [hunk 1]
```diff
@@ -1,7 +1,52 @@
-import React from "react";
+import React, { useState } from "react";
 import Filter from "filter";
 import FilterData from "./data.json";
-
+let data = {};
 export default function App() {
+  const airport = [
```
Comment (iCargopoc) [inline filter/example/src/App.js:9]: Add proper comments, since it's managed by iCargo developer

File: filter/src/panel/SavedFilters.js [hunk 2]
```diff
@@ -24,12 +24,12 @@ const SavedFilters = (props) => {
   let keyValue = "";
   let savedFilters = localStorage.getItem("savedFilters");
   savedFilters = savedFilters ? JSON.parse(savedFilters) : [];
-  if (savedFilters.length > 5) {
-    savedFilters = savedFilters.slice(
-      savedFilters.length - 5,
-      savedFilters.length
-    );
-  }
+  // if (savedFilters.length > 5) {
```
Comment (iCargopoc) [inline filter/src/panel/SavedFilters.js]: remove commented codes

**Review by iCargopoc — APPROVED** [review]

**Review by iCargopoc — APPROVED** [review]

**Review by iCargopoc — APPROVED** [review]

File: filter/src/index.js [hunk 3]
```diff
@@ -590,6 +615,20 @@ export default function Filter(props) {
...
    */
   const deleteAutoCompleteElement = (item) => {
+    filterData.filter.forEach((it) => {
+      it.types.forEach((tip) => {
+        if (tip.name === item.type && item.name === it.name) {
+          tip.weight = 400;
+        }
+      });
+    });
+    // let nameArray = [];
```
Comment (iCargopoc) [inline filter/src/index.js:625]: remove commented lines

**Review by iCargopoc — APPROVED** [review]

File: filter/example/src/App.js (hunk 1 above)
Comment (djnadackal) [reply filter/example/src/App.js:9]: resolved

File: filter/src/panel/SavedFilters.js (hunk 2 above)
Comment (djnadackal) [reply filter/src/panel/SavedFilters.js]: resolved

**Pull request merged**